tests/parallel/ExportTimeseries.xml

```diff
@@ -17,7 +17,7 @@
     <provide-mesh name="A" />
     <write-data name="S" mesh="A" />
     <write-data name="V" mesh="A" />
-    <export:vtk directory="timeseries" />
+    <export:vtu directory="timeseries" />
```

Comment:
```suggestion
```

This can be removed completely as the tests is explicitly for parallel solvers.

Reply:
Thanks for the feedback. Resolved in the latest commit.